### PR TITLE
Allowing negative line numbers in head

### PR DESCRIPTION
**Pull request by moisesluza**
Matches the same functionality UNIX head command allowing negative number of lines.
```
-n, --lines=[-]NUM       print the first NUM lines instead of the first 10;
                         with the leading '-', print all but the last
                         NUM lines of each file
```

### CODE REVIEW

File: tailhead/__init__.py [hunk 1]
```diff
@@ -223,11 +223,17 @@ def head(self, lines=10):
         """
         Return the top lines of the file.
```
Comment (Kentzo) [inline tailhead/__init__.py:224]: Could you also expend the docstring to document new behavior?

Comment (moisesluza) [reply tailhead/__init__.py:224]: Oops, I forgot about documentation. I added it here and in the user interface below. Also added a new example in README.rst

Comment (moisesluza) [reply tailhead/__init__.py:224]: I have added the documentation you suggested and an example in README.rst. Please take a look.

Comment (moisesluza) [reply tailhead/__init__.py:224]: @Kentzo could you please take a look at the documentation added?

**Comment by Kentzo** [comment]
@moisesluza Thank you!

**Pull request merged**